tests/functional/tests.py

```diff
@@ -1891,6 +1891,32 @@ def test_upload_snowball_objects(log_entry):
         _CLIENT.remove_object(bucket_name, "my-object3")
         _CLIENT.remove_bucket(bucket_name)
 
+    # run with staging file option
```

Comment:
Have `def _test_upload_snowball_objects(log_entry, staging_filename):` and use the method to have two tests named `test_upload_snowball_objects()` and `test_upload_snowball_objects_staging_file` to avoid code duplication.